semanticdb-javac/src/main/java/com/sourcegraph/semanticdb_javac/SemanticdbVisitor.java

```diff
@@ -221,9 +221,14 @@ public Void visitMemberSelect(MemberSelectTree node, Void unused) {
   public Void visitNewClass(NewClassTree node, Void unused) {
     if (node instanceof JCTree.JCNewClass) {
       JCTree.JCNewClass cls = (JCTree.JCNewClass) node;
-      emitSymbolOccurrence(cls.constructor, cls, Role.REFERENCE, CompilerRange.FROM_START_TO_END);
+      emitSymbolOccurrence(cls.constructor, cls, Role.REFERENCE, CompilerRange.FROM_TEXT_SEARCH);
```

Comment:
Can we actually pass in the identifier to avoid text search?
```suggestion
      emitSymbolOccurrence(cls.constructor, cls.getIdentifier(), Role.REFERENCE, CompilerRange.FROM_START_TO_END);
```

Reply:
That seems to also include the type parameters interestingly :thinking: 
![image](https://user-images.githubusercontent.com/18282288/114876309-1d0f7500-9df6-11eb-91c2-478397a55059.png)

Reply:
And potentially erroneous results :grimacing: 

![image](https://user-images.githubusercontent.com/18282288/114876370-2a2c6400-9df6-11eb-81d1-d3380ae3e9fb.png)

Reply:
Interesting, text search seems to be the way to go then.